### PR TITLE
Develop into master

### PR DESCRIPTION
**Pull request by dasunpubudumal**
Closes #

#### Changes proposed in this pull request

- Adding the RVI prefix into the configuration

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_  
_[Production Merges to `main`]_  
 &nbsp; &nbsp; \- _Check story numbers included_  
 &nbsp; &nbsp; \- _Check for debug code_  
 &nbsp; &nbsp; \- _Check version_

### CODE REVIEW

File: .github/workflows/check_release_version.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
 
       - name: Get specific changed files
         id: changed-files-specific
-        uses: tj-actions/changed-files@v23
+        uses: tj-actions/changed-files@v41
```
Comment (sdjmchattie) [inline .github/workflows/check_release_version.yml:17]: What happens if you don't update this version number?  All the repos in PSD use this check for GitHub pipelines.

**Review by yoldas — APPROVED** [review]
Looks good.

File: .github/workflows/check_release_version.yml (hunk 1 above)
Comment (yoldas) [reply .github/workflows/check_release_version.yml:17]: I have updated several repos for this. It is a security update.  https://www.cvedetails.com/cve/CVE-2023-51664/#

Comment (sdjmchattie) [reply .github/workflows/check_release_version.yml:17]: Sounds good @yoldas .  I guess I would suggest that you document which ones you've updated and create a task to update all the remaining ones.

Comment (yoldas) [reply .github/workflows/check_release_version.yml:17]: I have now checked out all repos and inspected the tj-actions/changed-files lines using a simple script. All repos that use the GitHub action have their version already updated to v41 except automanual (readonly repo at v23). 
As far as I remember, the patch came from dependabot and was applied and we also manually fixed some repos.

**Pull request merged**